src/lib.rs

```diff
@@ -98,15 +98,86 @@ impl KmerCountTable {
         hash_keys.iter().map(|&key| self.get_hash(key)).collect()
     }
 
-    // TODO: Add method "drop"
-    // remove kmer from table
+    /// Drop a k-mer from the count table by its string representation
+    pub fn drop(&mut self, kmer: String) -> PyResult<()> {
+        // Ensure that the k-mer length matches the table's ksize
+        if kmer.len() as u8 != self.ksize {
+            // Return an error if the lengths do not match
+            Err(PyValueError::new_err(
+                "kmer size does not match count table ksize",
+            ))
+        } else {
+            // Compute the hash of the k-mer using the same method used for counting
+            let hashval = self.hash_kmer(kmer).unwrap();
```

Comment:
you can replace the if/else with just

```
let hashval = self.hash_kmer(kmer)?;
```
which will forward the `Err` up the call stack. (I know I didn't do that elsewhere, not sure why ;)

Reply:
That makes sense. Done.